homeassistant/components/snapcast/media_player.py

```diff
@@ -258,7 +258,7 @@ def state(self):
         """Return the state of the player."""
         if self._client.connected:
             return STATE_ON
-        return STATE_OFF
+        return STATE_IDLE
```

Comment:
I think this should rather be STANDBY. IDLE is more for the case when it's on and running, but doing nothing.

Reply:
Makes sense, I adjusted the state. Wasn't aware of STATE_STANDBY, just getting startet with homeassistant.
Thanks for pointing it out!